### DISCUSSION TITLE
在OneDrive在微软云添加内容 结果无法添加 我也不知道什么原因

### DISCUSSION THREAD

**Discussion by ghost** [AlistGo/alist#6795] [Invalid question]
![image](https://github.com/user-attachments/assets/c9069cdb-4dff-48e9-bb04-d4ca0e14d4de)
显示已弃用我也没办法添加云盘了

**Comment by Simonzhang8** [comment]
直接用alist提供的client id和secret，不需要自己创建，因为现在微软限制了创建应用程序的条件
![image](https://github.com/user-attachments/assets/7ea509ef-a1b3-477b-893e-e802e634f086)

**Reply by ghost** [reply]
哦 我知道了 先用你的吧 在写你的建议别忘了你的网站
